src/util/api.js

```diff
@@ -98,3 +98,15 @@ export const initiatePrivileged = body => {
 export const transitionPrivileged = body => {
   return post('/api/transition-privileged', body);
 };
+
+// Create user with identity provider (e.g. Facebook or Google)
+//
+// If the authentication with idp fails, we will create a new user.
+// For that user needs to confirm data fetched from the idp.
+// This endpoint creates the new user with confirmed data.
```

Comment:
Is this correct? So if the user can not authenticate to th IdP, we'll create a user? Or if the authentication succeeds, we'll create a user with confirmed data from the IdP.